### PR TITLE
Initial support for minikube and kubernetes in micronaut-webapp.

### PR DESCRIPTION
**Pull request by tzezula**

### CODE REVIEW

**Comment by tzezula** [comment]
@eginez can you review the changes. Thanks!

File: micronaut-webapp/build-docker-compose.sh [hunk 1]
```diff
@@ -28,7 +28,7 @@ services:
     ports:
       - 8081:8081
     environment:
-      - TODOSERVICE_URL=http://todo-service:8080
+      - TODOSERVICE_URL=http://todo-service:8443
```
Comment (eginez) [inline micronaut-webapp/build-docker-compose.sh]: https here probably right?
```suggestion
      - TODOSERVICE_URL=https://todo-service:8443
```

File: micronaut-webapp/frontend/docker-build.sh [hunk 2]
```diff
@@ -1,6 +1,13 @@
 #!/bin/sh
 image_name=frontend-native
 docker_file=Dockerfile
+quiet=false
```
Comment (eginez) [inline micronaut-webapp/frontend/docker-build.sh]: thanks for the flag! 🎉

File: micronaut-webapp/build-kubernetes.sh [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+        initContainers:
+        - name: init-todo-loadtest-1
+          image: alpine
+          volumeMounts:
+          - mountPath: /opt/loadTest
+            name: data
+          command: ['sh', '-c', 'apk add git; git clone https://github.com/graalvm/graalvm-demos.git; cp graalvm-demos/micronaut-webapp/loadTests/* /opt/loadTest; rm -rf graalvm-demos']
+        - name: init-todo-loadtest-2
+          image: alpine
+          command: ['sh', '-c', 'until echo "GET /" | nc  $TODO_SERVICE_SERVICE_HOST $TODO_SERVICE_SERVICE_PORT; do echo waiting for NoSQL...; sleep 1; done;']
```
Comment (eginez) [inline micronaut-webapp/build-kubernetes.sh]: small name change in message
```suggestion
          command: ['sh', '-c', 'until echo "GET /" | nc  $TODO_SERVICE_SERVICE_HOST $TODO_SERVICE_SERVICE_PORT; do echo waiting for Todo service...; sleep 1; done;']
```

File: micronaut-webapp/build-docker-compose.sh (hunk 1 above)
Comment (tzezula) [reply micronaut-webapp/build-docker-compose.sh]: Fixed

File: micronaut-webapp/build-kubernetes.sh (hunk 3 above)
Comment (tzezula) [reply micronaut-webapp/build-kubernetes.sh]: Thanks!
I am always copying the pattern and forget to change the service name. :-)
Fixed.

**Comment by tzezula** [comment]
@eginez thanks for the review!
@shelajev can you merge this Pull Request. Thanks!

**Pull request merged**